dbt_platform_helper/utils/git.py

```diff
@@ -6,10 +6,13 @@ def git_remote():
     git_repo = subprocess.run(
         ["git", "remote", "get-url", "origin"], capture_output=True, text=True
     ).stdout.strip()
+    return extract_repository_name(git_repo)
```

Comment:
Is there value in adding a test to verify that `git_remote()` calls `extract_repository_name()`?

Or does some other test fail in a way that makes the problem obvious if it's delated?

Reply:
I couldn't work out what I needed to do for "@patch" to make that work. 
What I'd *love* to do is look at where this is called and see if we can move the subprocess.run bit out of this entirely. Or even fetch the github url from somewhere else entirely?